.github/workflows/ci_release.yml

```diff
@@ -4,6 +4,9 @@ on:
   push:
     branches:
       - main
+      - '**' # This allows the workflow to trigger on any branch
+    tags:
+      - 'v*.*.*'
```

Comment:
I think you were looking for something else https://regex101.com/r/GO0uEm/1

Maybe just `v.*` ? as in https://github.com/celestiaorg/celestia-node/pull/3787 (shiiit, should be `v.*` in the mentioned PR 😬 )